src/data/bundle.lean

```diff
@@ -0,0 +1,60 @@
+/-
+Copyright © 2021 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nicolò Cavalleri
+-/
+
+import tactic.basic
+import algebra.module.basic
+
+/-!
+# Bundle
+Basic data structure to implement fiber bundles, vector bundles (maybe fibrations?), etc. This file
+should contain all possible results that do not involve any topology.
+We provide a type synonym of `Σ x, E x` as `bundle.total_space E`, to be able to endow it with
+a topology which is not the disjoint union topology. In general, the constructions of fiber bundles
```

Comment:
Can you include the name of the "disjoint union topology" in backticks so that it links?